packages/flutter_tools/lib/src/asset.dart

```diff
@@ -138,7 +144,13 @@ class _ManifestAssetBundleFactory implements AssetBundleFactory {
   final bool _splitDeferredAssets;
 
   @override
-  AssetBundle createBundle() => ManifestAssetBundle(logger: _logger, fileSystem: _fileSystem, platform: _platform, splitDeferredAssets: _splitDeferredAssets);
+  AssetBundle createBundle() => ManifestAssetBundle(
+        logger: _logger,
```

Comment:
nit. this should be indented previous indent + 2: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#indent-multi-line-argument-and-parameter-lists-by-2-characters

Reply:
Yeah, I didn't check the autoformatter's work here